composer.json

```diff
@@ -1,159 +1,116 @@
-    "drupal/manage_display": "^1.0@alpha",
-    "drupal/menu_admin_per_menu": "^1.0",
-    "drupal/menu_link_attributes": "^1.0",
-    "drupal/page_manager": "^4.0@beta",
-    "drupal/panelbutton": "^1",
-    "drupal/panels_everywhere": "^4.0@beta",
-    "drupal/paragraphs": "^1.9",
-    "drupal/pathauto": "^1.6",
-    "drupal/rabbit_hole": "^1.0@beta",
-    "drupal/redirect": "^1.4",
-    "drupal/styleguide": "^1",
-    "drupal/svg_image": "^1.9",
-    "drupal/swiftmailer": "^1",
-    "drupal/telephone_validation": "^2.2",
-    "drupal/token": "^1",
-    "drupal/username_enumeration_prevention": "^1.0",
-    "drupal/views_bulk_edit": "^2.3",
-    "drupal/webform": "^5.4",
-    "drush/drush": "^10.1.0",
-    "webflo/drupal-finder": "^1.1",
-    "zaporylie/composer-drupal-optimizations": "^1.0"
-  },
-  "require-dev": {
-    "dmore/behat-chrome-extension": "^1.3",
-    "drupal/config_inspector": "^1",
-    "drupal/devel": "3.x-dev",
-    "drupal/drupal-extension": "^3.4",
-    "espend/behat-placeholder-extension": "^1.1",
-    "genesis/behat-fail-aid": "^2.1",
-    "mglaman/drupal-check": "^1.1"
-  },
-  "conflict": {
-    "drupal/drupal": "*"
-  },
-  "minimum-stability": "dev",
-  "prefer-stable": true,
-  "config": {
-    "sort-packages": true
-  },
-  "autoload": {
-    "classmap": [
-      "scripts/composer/ScriptHandler.php"
-    ]
-  },
-  "scripts": {
-    "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
-    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
-    "list-scaffold-files": [
-      "SkilldDrupal\\composer\\ScriptHandler::listScaffoldFiles"
     ],
-    "pre-install-cmd": [
-      "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "ckeditor/panelbutton",
+                "version": "4.10.1",
+                "type": "drupal-library",
+                "dist": {
+                    "url": "https://download.ckeditor.com/panelbutton/releases/panelbutton_4.10.1.zip",
+                    "type": "zip"
+                }
+            }
+        }
     ],
-    "pre-update-cmd": [
-      "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
-    ]
-  },
-  "extra": {
-    "installer-paths": {
-      "web/core": ["type:drupal-core"],
-      "web/libraries/{$name}": ["type:drupal-library"],
-      "web/modules/contrib/{$name}": ["type:drupal-module"],
-      "web/profiles/contrib/{$name}": ["type:drupal-profile"],
-      "web/themes/contrib/{$name}": ["type:drupal-theme"],
-      "drush/contrib/{$name}": ["type:drupal-drush"]
+    "require": {
+        "ckeditor/panelbutton": "^4.10",
+        "cweagans/composer-patches": "^1.6.5",
+        "composer/installers": "^1.2",
+        "drupal/core-composer-scaffold": "^9",
```

Comment:
it should not be used

Reply:
As you can see at https://packagist.org/packages/drupal/core-recommended#9.0.0-beta2
dependencies already require `composer/installers: v1.8.0` and message is unused